example.tex

```diff
@@ -20,7 +20,7 @@
 
 \usepackage[utf8]{inputenc}
 \usepackage{lipsum}
-\usepackage[bg-letter]{dnd} % Options: bg-a4, bg-letter, bg-full, bg-print, bg-none, justified
+\usepackage{dnd} % Options: bg-full*, bg-print, bg-none, justified
```

Comment:
What is the asterisk for?

Reply:
Marks it as a default option. I guess I should make that more clear and use something else. Especially as asterix means something in LaTeX.